foreman/data_refinery_foreman/surveyor/test_surveyor.py

```diff
@@ -26,7 +26,7 @@ def test_calls_survey(self, survey_method):
         job.save()
         surveyor.run_job(job)
 
-        survey_method.assert_called_with(job)
+        survey_method.assert_called_with()
```

Comment:
Hmmm, now that this method isn't taking this argument, I'm wondering what exactly it is doing. Should it be called ``assert_called_with_job`` or something of the sort? Also, where is it defined again?